apis/duck/v1alpha1/condition_set.go

```diff
@@ -212,11 +212,16 @@ func (r conditionsImpl) SetCondition(new Condition) {
 }
 
 func (r conditionsImpl) isTerminal(t ConditionType) bool {
-	for _, cond := range append(r.dependents, r.happy) {
+	for _, cond := range r.dependents {
```

Comment:
side note. I think the name of this function is confusing... not your doing...

I thought append returned the new slice so is this really any different from what was there before?

Reply:
oops, I started reviewing, went back to the golang docs, and re-educated myself about append happiness, and forgot to get rid of this comment :)